docs/src/components/ComponentPage/AndroidComponentPage.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import BpkLink from 'bpk-component-link';
+
+import ComponentScreenshots from '../DocsPageBuilder/ComponentScreenshots';
+
+import ComponentPage from './ComponentPage';
+
+type Props = {
+  documentationId: string,
+  readme: string,
+  screenshots: [
+    {
+      id: string,
+      screenshots: {
+        altText: string,
+        height: number,
+        src: string,
+        subText: string,
+        title: string,
+        width: number,
+      },
+      title: string,
+    },
+  ],
+  usageTable: ?{
+    dos: Array<string>,
+    donts: Array<string>,
+  },
+};
+
+const AndroidComponentPage = (props: Props) => {
+  const { documentationId, screenshots, readme, usageTable } = props;
```

Comment:
Can we spread `...rest` on to the outer ComponentPage instead of repeating things?

Reply:
I originally did do this, but then I was concerned it would violate principle of least surprise by not making it obvious that `readme` and `usageTable` were being passed through, you'd only know they were there from looking at the Flow type. 

If you feel strongly about it I don't mind changing it, but that was my rationale.

Reply:
I see it like `AndroidComponentPage` is effectively a subclass of `ComponentPage`, so it could be inferred, however explicit won't do any harm either.